src/Raven.Server/Smuggler/Documents/StreamSource.cs

```diff
@@ -1283,7 +1304,7 @@ public static BlittableJsonReaderObject WriteDummyDocumentForAttachment(Document
             return context.ReadObject(djv, details.Id);
         }
 
-        private async IAsyncEnumerable<DocumentItem> ReadDocumentsAsync(INewDocumentActions actions = null)
+        private async IAsyncEnumerable<DocumentItem> ReadDocumentsAsync(List<string> collectionsToOperate, INewDocumentActions actions = null)
```

Comment:
[Yes](https://github.com/ravendb/ravendb/pull/15383#discussion_r1042394452), all that we can filter on export, we filter on import (except counters, see below).
- **TimeSeries** - done with test coverage.
- **Attachments** will be filtered inside the `DocumentRead` method as a part of a filtered document.
- **Counters** we can't filter because it has no information about the collection inside. But all counters for Documents that have not been filtered are not imported (test approved it).
- **Tombstones** - done with test coverage.
- **Conflicts** - done with test coverage.